src/asm/symbol.c

```diff
@@ -276,6 +276,13 @@ static bool isReferenced(struct Symbol const *sym)
 	return sym->ID != (uint32_t)-1;
 }
 
+struct SymbolListNode {
+	struct Symbol *this;
```

Comment:
:pick: C++ is close enough to C that it feels weird to use `this` as a variable. (Same goes for `class`, etc.)

Reply:
Only because your highlighter is reminding you of it :P
I use C++ keywords as C identifiers all the time (`new` is a common one) and I've never seen any issues come out of that, even when building with C++-capable compilers (GCC, Clang, etc.). The toolchains don't seem to be confused.

Reply:
I didn't say "it could break", I said "it feels weird". :P And we *were* considering to convert things gradually to C++, although the Rust rewrite probably superseded that.

Reply:
I felt a little bit weird doing it, but I also felt free! (will use `sym`, thanks)